CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
 - Add `Monitoring/Alarms/Enabled` parameter to toggle Amazon CloudWatch Alarms for the cluster.
 - Add head node alarms to monitor EC2 health checks, CPU utilization and the overall status of the head node.
 - Add the option to use EFS storage instead of NFS exports from the head node root volume for intra-cluster shared ParallelCluster, Intel, Slurm, and login node data.
+- Add the option to use EFS or FSx as external shared storage via the `SharedStorage` section of the config file for `/home`
```

Comment:
I'd say "Allow to use" rather than "Add the option" because "Add the option" makes me think that we modified something in the cluster config schema.

Reply:
Ah good point